include/dali/core/span.h

```diff
@@ -218,10 +218,10 @@ DALI_HOST_DEV constexpr span<T, N> make_span(T (&a)[N]) {
 }
 
 template <span_extent_t Extent, typename T>
-DALI_HOST_DEV constexpr span<const T, Extent> make_cspan(T *data) { return { data }; }
+DALI_HOST_DEV constexpr span<const T, Extent> make_cspan(const T *data) { return { data }; }
```

Comment:
Why this particular change? It worked fine before.

Reply:
If it's not for this `const`, `int [2]` falls into `make_cspan(Collection)` instead of `make_cspan(const T(&a)[N]`. Dunno why...
Build error without this `const`:
```
/Dali/include/dali/core/span.h: In instantiation of ‘constexpr auto dali::make_cspan(Collection&) [with Collection = int [2]]’:
/Dali/dali/kernels/reduce/reduce_cpu_test.cc:60:40:   required from here
/Dali/include/dali/core/span.h:231:23: error: request for member ‘data’ in ‘c’, which is of non-class type ‘int [2]’
   return make_cspan(c.data(), c.size());
                     ~~^~~~
Dali/include/dali/core/span.h:231:33: error: request for member ‘size’ in ‘c’, which is of non-class type ‘int [2]’
   return make_cspan(c.data(), c.size());
                               ~~^~~~
/Dali/dali/kernels/reduce/reduce_cpu_test.cc:60:41: error: invalid use of void expression
   mean.Setup(out3, in, make_cspan(axes3));
```